errors/errors.go

```diff
@@ -0,0 +1,276 @@
+package errors
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+// TritonError represents an error code and message along with
+// the status code of the HTTP request which resulted in the error
+// message. Error codes used by the Triton API are listed at
+// https://apidocs.joyent.com/cloudapi/#cloudapi-http-responses
+type TritonError struct {
```

Comment:
Can we do a `go-rename` on `TritonError` and change it to be just `Triton` ?  The struct type is already scoped to the package name.  Future type assertions would read: `errors.Triton` or however the consumer aliases `triton-go/errors`.  ?